src/components/PrimaryPanes/SourcesTree.js

```diff
@@ -301,6 +301,10 @@ class SourcesTree extends Component<Props, State> {
         className={classnames("arrow", {
           expanded: expanded
         })}
+        onClick={e => {
```

Comment:
Since we're allowing click on the entire tree node, I think we can remove this all together -- the icon at this point is nothing but a UI hint.

Reply:
this is what fixed it... :)